src/util/email.js

```diff
@@ -532,12 +534,31 @@ module.exports = (formio) => {
               );
             }));
           }, Promise.resolve());
-        })
-        .then((response) => next(null, response))
-        .catch((err) => {
-          debug.error(err);
-          return next(err);
         });
+
+        if (req.user) {
+          return sendEmails()
+            .then((response) => next(null, response))
+            .catch((err) => {
+              debug.error(err);
+              return next(err);
+            });
+        }
+     else {
+       // direct the sending of emails without user parameter to non-priority tasks queue
+        const sendWithTimeout = () => {
+           setTimeout(()=>{
```

Comment:
I don't believe this will do what you think it does. If I hit the server 10 times per/second, it seems that this will still send emails at 10 times per/second, but the start of them will just delay 10 seconds. This does not achieve what we are after. We need to instead perform a throttle.

I am thinking that the Lodash throttle method would be sufficient here.

https://lodash.com/docs/4.17.15#throttle